Adjust the ENOBUFS syslog level on the Netlink routing to LOG_WARNING.

From: Arun Barboza <arun.barboza@broadcom.com>


---
 ntpd/ntp_io.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: b/ntpd/ntp_io.c
===================================================================
--- a/ntpd/ntp_io.c
+++ b/ntpd/ntp_io.c
@@ -4709,7 +4709,7 @@ process_routing_msgs(struct asyncio_read
 
 	if (cnt < 0) {
 		if (errno == ENOBUFS) {
-			msyslog(LOG_ERR,
+			msyslog(LOG_WARNING,
 				"routing socket reports: %m");
 		} else {
 			msyslog(LOG_ERR,
